lib/puppet_litmus/serverspec.rb

```diff
@@ -158,4 +158,29 @@ def run_bolt_task(task_name, params = {})
     yield result if block_given?
     result
   end
+
+  # Runs a script against the target system.
+  #
+  # @param script [String] The path to the script on the source machine
+  # @param opts [Hash] Alters the behaviour of the command. Valid options are :expect_failures [Boolean] doesnt return an exit code of non-zero if the command failed.
+  # @yieldreturn [Block] this method will yield to a block of code passed by the caller; this can be used for additional validation, etc.
+  # @return [Object] A result object from the script run.
+  def bolt_run_script(script, opts = {})
+    target_node_name = ENV['TARGET_HOST'] if target_node_name.nil?
+    inventory_hash = if target_node_name.nil? || target_node_name == 'localhost'
+                       nil
+                     else
+                       inventory_hash_from_inventory_file
+                     end
+
+    result = run_script(script, target_node_name, options: opts, config: nil, inventory: inventory_hash)
```

Comment:
There's also a parameter for arguments: https://github.com/puppetlabs/bolt/blob/master/lib/bolt_spec/run.rb#L66

Reply:
@florindragos Good spot. I'll add in an arguments parameter.

Reply:
@florindragos Now updated with arguments.